electron/electron/handlers/runWhisper/runWhisper.ts

```diff
@@ -86,21 +86,45 @@ export default ipcMain.handle(
     if (device) inputArray.push('--device', device);
 
     // Add the input path
-    inputArray.push(inputPath);
+    inputArray.push('--input', inputPath);
 
     // ---------------------------------  Run the whisper script --------------------------------- //
 
     console.log('RunWhisper: Running model with args', inputArray);
 
     // Spawn the whisper script
-    const childProcess = spawn('whisper', inputArray, { stdio: 'inherit' });
+    let childProcessArgs: [string, string[]?, object?];
+    if (isDev) {
+      const backendDir = join(__dirname, '../../../../backend/');
+      childProcessArgs = [
+        'poetry',
+        ['run', 'stagewhisper', ...inputArray],
+        {
+          stdio: 'inherit',
+          cwd: backendDir
+        }
+      ];
+    } else {
+      childProcessArgs = ['echo', ['"Production backend not yet implemented"']];
+    }
+    const childProcess = spawn(...childProcessArgs);
 
     const transcription = await new Promise<Transcription>((resolve, reject) => {
-      childProcess.on('data', (data: string) => {
+      childProcess.on('error', (error: Error) => {
+        console.log(`RunWhisper: Failed to start subprocess: ${error}`);
+      });
+
+      // If the stdio/out/err streams are available, read them to the console
+      // Note: if `stdio` is set to anything but `"pipe"` then these will be undefined
+      // and these events will never fire.
+      childProcess.stdout?.on('data', (data: string) => {
         console.log(`stdout: ${data}`);
       });
-      childProcess.on('error', (error: Error) => {
```

Comment:
I believe the error event actually indicates that the process failed to start, not that a message was received over stderr. I've moved this event handler above the stdio ones and rejiggered the message.